WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -2041,6 +2079,15 @@ - (void)editorDidPressMedia:(WPEditorViewController *)editorController
                                       [weakSelf showMediaPickerForWordPressLibraryAnimated:YES];
                                   }];
 
+    if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 11.0 &&
```

Comment:
I believe that in Xcode 9 you can now write this:

```if (@available(iOS 11, *)) {}```

Reply:
@SergioEstevao isn't that Swift? I'm not seeing anything akin to that for ObjC.

Reply:
give it a try :)

Reply:
Well, I did. :) But I'll try again.

Reply:
Nice hint @SergioEstevao.  Just to confirm... it's working for me as well.